chore: Update translations #36

ymind opened the pull request:
Close #36

lppedd commented:
Hey! Thanks a lot :smile:

lppedd commented:
I think there are a couple of messages to align, which maybe you didn't notice.

```
cc.config.info=Perform commit completion using:
cc.config.types=Types:
cc.config.scopes=Scopes:
cc.config.coAuthorsDialog.error=Co-author's names cannot be blank
```
And btw, feel free to omit colons or whatever if it's not appropriate for Chinese.

ymind commented:
Thank you for noting this, I've re-checked all entries and corrected them.

I deliberately omitted the colon for the following two entries:
```properties
cc.config.types=Types:
cc.config.scopes=Scopes:
```
Most times the colon in the title is not necessary in Chinese.

I also fixed the translation of the following entries:
```properties
cc.config.providersPriority=Personalize the providers priority (higher in the list equals higher priority).
cc.config.providersPriority=自定义提供程序优先级 (越靠前优先级越高)。
```
This may be caused by an incorrect copy and paste.

----

If you like or studying Chinese, I admire it! As far as I know, Chinese is very difficult for most foreigners.
I am not a professional language teacher, but if I can help you, I will be very pleased!

lppedd commented:
Awesome, thanks for the update!
I'll merge tonight after pushing the latest code.

I have a question regarding properties like

lppedd commented:
Awesome, thanks for the update!
I'll merge tonight after pushing the latest code.

I have a question regarding properties like
```
cc.whatsnew.dialog.newer=&Newer
```
In the standard file I've used the `&` character to let a user use it's a keyboard shortcut (`ALT+N` in this case).
How does it work with Chinese?

ymind commented:
About keyboard shortcuts, the general approach is:
```properties
cc.whatsnew.dialog.newer=&Newer
cc.whatsnew.dialog.newer=新版本(&N)
```
This will be shown as:
![image](https://user-images.githubusercontent.com/315716/98456967-922eac80-21bd-11eb-9259-54f7da42b45a.png)

BUT, I didn't find a way to visit the GUI during the test, so I omitted it temporarily. I'm not sure if `新版本(&N)` will disturb with the UI display.

----

The translation has been updated.

lppedd commented:
Thanks! No problem for the shortcuts, you can add them later on of you want.

To test that UI part you should use the What's new in Conventional Commit action, under help, though it's empty so buttons will be grayed out.

lppedd commented:
Last added message before relase, I promise 😄 

```
cc.config.filePicker.error.existence=The file does not exist
```

ymind commented:
> ![image](https://user-images.githubusercontent.com/19871649/98465413-310cd600-21c9-11eb-8e70-9f9ca0719098.png)

I find it!

Comprehensive screenshot and source code, I understand the `newer` and `older` here as `next` and `previous`. There are two way translations:

A
```properties
# direct translation
cc.whatsnew.dialog.newer=较新 (newer)
cc.whatsnew.dialog.older=较旧 (older)
```

B
```properties
# integrated with Chinese customary translation
cc.whatsnew.dialog.newer=下一版 (next)
cc.whatsnew.dialog.older=上一版 (previous)
```

**I'm using B now, please confirm !**

And the `Close` button text has no translate entity. Is this due to IDEA API restrictions?

lppedd commented:
Yes, newer and older are used as alias for next and previous (there are new releases and old releases so it seemed appropriate).

I'd say: go with *B* if you've already seen it used in other programs, otherwise go with *A*. 

I could probably make the Close button translatable, let me check.

lppedd commented:
Ok, the new property is
```
cc.whatsnew.dialog.close=&Close
```

lppedd commented:
And I have a question. Many localized strings ends with the ideaographic full stop charachter.  
Does it represent a simple dot (like in english) or does it have a special role in the sentence?
If it's a simple dot I would remove it from some places, per [UI guidelines](https://jetbrains.design/intellij/text/punctuation/#period), let me know 😄 

![image](https://user-images.githubusercontent.com/19871649/98525559-eb0f4b00-2278-11eb-8af0-faf26c315f48.png)

ymind commented:
> ![image](https://user-images.githubusercontent.com/19871649/98525559-eb0f4b00-2278-11eb-8af0-faf26c315f48.png)

This is a full stop in Chinese. Usage is equivalent to english '.'.

In this commit, I removed them.

----
**In this translation work, you let me learn a lot of things, thank you very much!**

lppedd commented:
@ymind thanks a lot to you! Merged successfully.
